libkbfs/config_local.go

```diff
@@ -777,8 +777,9 @@ func (c *ConfigLocal) EnableJournaling(journalRoot string) {
 	// it if it doesn't exist, make sure that it doesn't
 	// point to /keybase itself, etc.
 	log := c.MakeLogger("")
+	onBranchChange := c.KBFSOps().(*KBFSOpsStandard)
```

Comment:
seems more robust to do:

```
listener, _ := c.KBFSOps().(branchChangeListener)
...makeJournalServer(..., listener)
```

in case `KBFSOps` is swapped out in a test. Although I'm fine leaving it as is until we need it.